.vscode/launch.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
@iTOYS how do you think, should we add this line to gitignore?

Reply:
I guess this is helpful template for contributors.

Reply:
Yes, agreed, but we could use it as well.